diff --git a/packages/contracts-bedrock/interfaces/dispute/IInitializable.sol b/packages/contracts-bedrock/interfaces/dispute/IInitializable.sol
index a3bb74b..bc998e1 100644
--- a/packages/contracts-bedrock/interfaces/dispute/IInitializable.sol
+++ b/packages/contracts-bedrock/interfaces/dispute/IInitializable.sol
@@ -2,5 +2,5 @@
 pragma solidity ^0.8.0;
 
 interface IInitializable {
-    function initialize() external payable;
+    function initialize(bytes calldata initData) external payable;
 }
diff --git a/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol b/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol
index eb92ca5..c3521f7 100644
--- a/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol
+++ b/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol
@@ -143,11 +143,13 @@ contract DisputeGameFactory is ProxyAdminOwnedBase, ReinitializableBase, Ownable
     /// @param _gameType The type of the DisputeGame - used to decide the proxy implementation.
     /// @param _rootClaim The root claim of the DisputeGame.
     /// @param _extraData Any extra data that should be provided to the created dispute game.
+    /// @param initData The initialization data for the DisputeGame.
     /// @return proxy_ The address of the created DisputeGame proxy.
     function create(
         GameType _gameType,
         Claim _rootClaim,
-        bytes calldata _extraData
+        bytes calldata _extraData,
+        bytes calldata initData
     )
         external
         payable
@@ -198,7 +200,7 @@ contract DisputeGameFactory is ProxyAdminOwnedBase, ReinitializableBase, Ownable
                 )
             );
         }
-        proxy_.initialize{ value: msg.value }();
+        proxy_.initialize{value: msg.value}(initData);
 
         // Compute the unique identifier for the dispute game.
         Hash uuid = getGameUUID(_gameType, _rootClaim, _extraData);
